src/main/java/world/bentobox/bentobox/util/Util.java

```diff
@@ -538,6 +543,44 @@ private static boolean isJUnitTest() {
         return false;
     }
 
+
+    /**
+     * This method translates color codes in given string and strips whitespace after them.
+     * This code parses both: hex and old color codes.
+     * @param textToColor Text which color codes must be parsed.
+     * @return String text with parsed colors and stripped whitespaces after them.
+     */
+    @NonNull
+    public static String translateColorCodes(@NonNull String textToColor) {
```

Comment:
This implementation is based on:
https://www.spigotmc.org/threads/hex-color-code-translate.449748/#post-3867804

The general idea is to pass a sequence of 6 color codes to the text parser. Another alternative would be to use BungeeChat directly and use direct hex color parsing:
https://github.com/SpigotMC/BungeeCord/blob/master/chat/src/main/java/net/md_5/bungee/api/ChatColor.java#L244-L274

It will produce the same result: `#ff0000` -> `§x§f§f§0§0§0§0` where `§x`  that it will be hex and sequence of 6 color codes will be its value.

The reason why I did not use BungeeChat ChatColor#of is because of the fact, that it does not support valid 3 letter hex codes: #0f2. That is shorten hex and it translates into #00ff22